packages/node_modules/@webex/plugin-meetings/src/metrics/mqa-processor.js

```diff
@@ -0,0 +1,110 @@
+import bowser from 'bowser';
+
+import {
+  STATS,
+  MQA_STATS
+} from '../constants';
+
+
+/**
+ * @description MQAProcessor handles interval data for MQA
+ * @export
+ * @class MQAProcessor
+ */
+class MQAProcessor {
+  /**
+     * @constructor
+     * @public
+     * @memberof Meetings
+     */
+  constructor() {
+    this.data = {
+      videoReceive: [],
+      audioTransmit: [],
+      audioReceive: [],
+      videoTransmit: []
+    };
+    this.intervalNumber = 1;
+  }
+
+  /**
+   * @param {String} id
+   * @param {Array} interval - a slice of metrics history
```

Comment:
Could you document the type of objects in the array?

Reference: https://jsdoc.app/tags-param.html#names-types-and-descriptions
```js
/**
 * Assign the project to a list of employees.
 * @param {Object[]} employees - The employees who are responsible for the project.
 * @param {string} employees[].name - The name of an employee.
 * @param {string} employees[].department - The employee's department.
 */
Project.prototype.assign = function(employees) {
    // ...
};
```